tensorflow_datasets/scripts/documentation/dataset_markdown_builder.py

```diff
@@ -0,0 +1,412 @@
+"""Dataset catalog documentation template.
+
+Displayed in https://www.tensorflow.org/datasets/catalog/.
+
+"""
+import dataclasses
+import functools
+import textwrap
+from typing import Callable, List
+
+import tensorflow_datasets as tfds
+
+
+@dataclasses.dataclass
+class Section(object):
+  get_signature: Callable[[tfds.core.DatasetBuilder], str]
+  make: Callable[[tfds.core.DatasetBuilder], str]
+
+# --------------------------- Builder sections ---------------------------
+
+
+def display_description(builder):
+  return textwrap.dedent(
```

Comment:
Is there a drawback of using tfds.core.utils.dedent everywhere, as it will strip extra whitespace in addition ?

Reply:
Yeah, it removes the required newlines